reo/scenario.py

```diff
@@ -317,29 +337,108 @@ def setup_pv(pv_dict, latitude, longitude, time_steps_per_hour):
+                    k_by_zone = copy.deepcopy(ghp.GHPGHXInputs.ground_k_by_climate_zone)
+                    climate_zone = get_climate_zone(ghpghx_post["latitude"], ghpghx_post["longitude"])
+                    ghpghx_post["ground_thermal_conductivity_btu_per_hr_ft_f"] = k_by_zone[climate_zone]
+                # Call /ghpghx endpoint to size GHP and GHX
+                ghpghx_post_resp = client.post('/v1/ghpghx/', data=ghpghx_post)
+                ghpghx_post_resp_dict = json.loads(ghpghx_post_resp.content)
+                ghp_uuid = ghpghx_post_resp_dict.get('ghp_uuid')
+                ghpghx_results_url = "/v1/ghpghx/"+ghp_uuid+"/results/"
+                ghpghx_results_resp = client.get(ghpghx_results_url)  # same as doing ghpMakeResponse(ghp_uuid)
+                ghpghx_results_resp_dict = json.loads(ghpghx_results_resp.content)
+                ghpghx_response_list.append(ghpghx_results_resp_dict)
+                #json.dump(ghpghx_response_list, open("ghpghx_response.json", "w"))
+                ghp_option_list.append(ghp.GHPGHX(dfm=dfm,
+                                                    response=ghpghx_response_list[i],
+                                                    **inputs_dict["Site"]["GHP"]))
+            # Update GHPModel with created ghpghx_response
+            tmp = dict()
+            tmp['ghpghx_response'] = ghpghx_response_list
+            ModelManager.updateModel('GHPModel', tmp, run_uuid)
+            # Sleep to avoid calling julia_api for /job (reopt) too quickly after /ghpghx (only required locally and/or for debugging?)
+            time.sleep(10)
+        # If ghpghx_response is included in inputs/POST, do NOT run /ghpghx model and use already-run ghpghx
+        elif inputs_dict["Site"]["GHP"].get("building_sqft") is not None and \
+                inputs_dict["Site"]["GHP"].get("ghpghx_response") not in [None, []]:
+            for i in range(len(inputs_dict["Site"]["GHP"]["ghpghx_response"])):
+                ghp_option_list.append(ghp.GHPGHX(dfm=dfm,
+                                                    response=inputs_dict["Site"]["GHP"]["ghpghx_response"][i],
+                                                    **inputs_dict["Site"]["GHP"]))
+        
         util = Util(dfm=dfm,
                     outage_start_time_step=inputs_dict['Site']['LoadProfile'].get("outage_start_time_step"),
                     outage_end_time_step=inputs_dict['Site']['LoadProfile'].get("outage_end_time_step"),
                     )
-                    
+
         dfm.add_soc_incentive = inputs_dict['add_soc_incentive']
 
+        if inputs_dict["Site"]["NewBoiler"]["max_mmbtu_per_hr"] > 0:
+            newboiler = NewBoiler(dfm=dfm, **inputs_dict['Site']['NewBoiler'])
+            # Any parameters processed and updated?
+            tmp = dict()
+            # Assign tmp["param"] = newboiler.xyx
+            ModelManager.updateModel('NewBoilerModel', tmp, run_uuid)
```

Comment:
Why update the model with an empty `dict`? (same for SteamTurbine).

Reply:
I removed this updateModel from both